### PR TITLE
ARGO-178 Metric details

### PR DESCRIPTION
**Pull request by pkoro**
- [x] Changes in XML
- [x] Usage of new routing
- [x] Changes in response logic
- [x] Changes in input

### CODE REVIEW

**Comment by pkoro** [comment]
please review @kaggis @skanct @makistsantekidis

File: doc/api/metric-result.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+### Input
+
+```
+/metric_result/{hostname}/{metric_name}?[exec_time]
+```
+
+#### Query Parameters
+
+Type            | Description                                                                                     | Required | Default value
```
Comment (skanct) [inline doc/api/metric-result.md]: Please make the table smaller and see how the tables are written in the documents for the compute engine

**Comment by makistsantekidis** [comment]
+1

**Comment by skanct** [comment]
I just noticed that we named this `metric_result`, while for everything else we use the `metricResult` notation

**Comment by pkoro** [comment]
Hmm, this is a good point. Even if this is something internal to the API we should have consistent naming across our packages. I can open a new PR to fix this if you want.

**Comment by skanct** [comment]
It is not just internal to the API:

/api/v2/metric_result/www.example.com/httpd_check?exec_time=2015-06-20T12:00:00Z

**Comment by pkoro** [comment]
Well this part is exactly how we have designed it in the document. (see pages 12,13 [here][l1]). I thought that you were referring to the internal package naming. 

[l1]: https://docs.google.com/document/d/13PoY75yKyDfI7Z6jTJGOgiApP0LQLO_XRKXzz5078U4/edit#

**Comment by skanct** [comment]
Yes I was referring to the internal naming. I guess it was used because the api call is metric_result. That was my point

**Comment by pkoro** [comment]
Ok, so we should change the internal package name to `metricResult` in order to be more consistent in that respect. I can this, no problem.

**Comment by pkoro** [comment]
Followed up in #118

**Pull request merged**